pkg/ingester/errors_test.go

```diff
@@ -237,6 +237,18 @@ func TestNewTSDBIngestExemplarErr(t *testing.T) {
 	checkIngesterError(t, wrappedErr, mimirpb.BAD_DATA, true)
 }
 
+func TestTooBusyError(t *testing.T) {
+	require.Error(t, tooBusyError)
+	require.Equal(t, tooBusyErrorMsg, tooBusyError.Error())
```

Comment:
This is a circular test (expecting the implementation), it's better to define the expectation in the test:

```suggestion
	require.Equal(t, "the ingester is currently too busy to process queries, try again later", tooBusyError.Error())
```